README.md

```diff
@@ -92,6 +92,15 @@ is easier to understand with an example:
               }
             , "expected"   : null
             }
+            { "description": "Foo'ing a very big number returns nothing"
+            , "optional"   : "big-ints"
+            , "comments"   : [ "Making this test case pass requires using BigInts." ]
```

Comment:
The example in #1492 explains in the `comments` that "Your track may choose to skip this test". That applies to all cases with an `optional`-key, so it doesn't need to be added to every case, imo. The comment outlines **why** the test is optional.